protocol-definitions/DynamicConfig.yaml

```diff
@@ -1427,7 +1427,7 @@ methods:
             List of resource definitions.
     response: {}
   - id: 20
-    name: removeNamespaceConfig
+    name: removeUserCodeNamespaceConfig
```

Comment:
I think the doc on the parameters needs to change as well

Reply:
Fixed this missing doc in https://github.com/hazelcast/hazelcast-client-protocol/pull/495/commits/5b1ae0e786997c22e4d1d837baa148dd03daef7e